test/queries_generator/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -x
+set -e
+
+g++ -o queries_generator queries_generator.cpp
```

Comment:
its binary utility (cpp sources), it should be a part of project cmake (not need for run.sh)
these utilities need several steps until completion, so its better to add help section to explain that.